packages/keyboard-shortcuts/src/store/actions.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Keyboard key combination.
+ *
+ * @typedef {Object} WPShortcutKeyCombination
+ *
+ * @property {string}  character Character.
+ * @property {string?} modifier  Modifier.
```

Comment:
If it's meant to be optional:

```suggestion
 * @property {string} [modifier]  Modifier.
```

See also:

- https://jsdoc.app/tags-type.html
- https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#nullable-undefined-and-void-types
- https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/#functions